ofctrl/ofAction.go

```diff
@@ -686,3 +688,31 @@ func (a *SetICMPv6CodeAction) GetActionMessage() openflow13.Action {
 func (a *SetICMPv6CodeAction) GetActionType() string {
 	return ActTypeSetICMP6Code
 }
+
+type SetICMPv4TypeAction struct {
+	Type uint8
+}
+
+func (a *SetICMPv4TypeAction) GetActionMessage() openflow13.Action {
+	field, _ := openflow13.FindFieldHeaderByName("NXM_OF_ICMP_TYPE", false)
```

Comment:
I notice `SetICMPv6TypeAction`'s `GetActionMessage` uses lower case for the field name, is it another typo or it's not case sensitive?

Reply:
Yes. The field name is not case sensitive. The FindFieldHeaderByName() will convert field name to upper automaticly.